### DISCUSSION TITLE
Tutorial about Python Packaging offering different use case (Request to Review)

### DISCUSSION THREAD

**Discussion by buhtz** [pypa/setuptools#4162] [Show and tell]
Hello,

I would like to point to my Python Packaging Tutorial explaining several  common use cases using minimal demo projects.

https://codeberg.org/buhtz/tech-demo-python-packaging

I am not an expert and assume that some of my solutions might not be the best. So I would appreciate if you can review them.

Background of that project: I am member of FOSS maintainer team and prepare the migration of [Back In Time](https://github.com/bit-team/backintime) from a makefile based build-system to a modern Python Build system using pyproject.toml & Co. To explore some expect able problems and possible solutions I created this minimal examples. I also do plan a tutorial repo about Debian Python Packaging using the same approach with minimal examples illustrating different use cases.

Thanks in advance,
Christian

PS: Posted this without response also on python-list@python.org and r/pythoncoding.

**Comment by webknjaz** [comment]
A lot of people who care about improving the learning materials in the context of packaging collaborate on PyPUG. Have you considered contributing there?

It's here: https://github.com/pypa/packaging.python.org

Also, a good place for such discussions (short of publishing to PyPUG) is https://discuss.python.org/c/packaging

P.S. `setuptools` is just one of the build backends nowadays. So posting here will probably have limited exposure.

**Comment by buhtz** [comment]
Thanks for this hint.
https://discuss.python.org/t/tutorial-about-python-packaging-offering-different-use-case-request-to-review/41285?u=buhtz
